server/src/main/java/org/spine3/server/entity/AbstractEntity.java

```diff
@@ -157,12 +160,126 @@ private static IllegalStateException noSuchConstructor(String entityClass, Strin
     }
 
     /**
+     * Obtains constructor for the passed aggregate part class.
+     *
+     * <p>The part class must have a constructor with ID and {@code AggregateRoot} parameters.
+     *
+     * @param entityClass the {@code AggregatePart} class
+     * @param idClass     the class of entity identifiers
+     * @param <E>         the entity type
+     * @param <I>         the ID type
+     * @return the {@code AggregatePart} constructor
+     * @throws IllegalStateException if the entity class does not have the required constructor
+     */
+    public static <E extends AggregatePart, I, R extends AggregateRoot<I>> Constructor<E>
```

Comment:
I think it is similar case with `getConstructor(Class<E> entityClass, Class<I> idClass)` method. And it is used outside the package in the `EntityBuilder` class and in the `AggregateRepository` class. If make it non-public and non-static we cant use it in the `testutil` module. And then it should be moved to the classes where it is used. Method is written by me is used in the `AggregatePartRepository` and `AggregatePartBuilder` classes.